src/02-common-principles.md

```diff
@@ -303,6 +303,30 @@ sub- prefix is not part of the subject label, but must be included in file names
 MUST be integers. Those labels MAY include zero padding, but this is NOT
 RECOMMENDED to maintain their uniqueness.
 
+## The contrast suffix
+
+In addition to the `*<label_identifier>-<custom_label>*` pairs (e.g. `sub-001`),
+it is RECOMMENDED that file names contain a suffix of the form `*_<suffix>.`,
+which directly precedes the file format delimiter. The string values of this
```

Comment:
Each data type has separate specific file naming rules. As far as I know there are no data types that allow something along the lines of `<key>-<value>.<ext>`.

Reply:
ok, so there is always a suffix, corresponding to no paritcular type of terms. But in the case of NIfTI files specifically, we can comment that the suffix specifies the contrast?

Reply:
Perhaps, but `phase1`, `phase1`, `magnitude1`, `defacemask` and other weird suffixes are being used for nifti files. Would "contrast" still cover them?

Reply:
yes, they would, though `defacemask` --- while it can certainly be conceptualized as a contrast --- is named in a way which does not inspire the concept of contrast. Just to give you an example, something along the lines of `anonymousanatomyregion` would do that. Still, unfortunate naming at most would require a sentence of clarificantion, it is in the end a feature which is being contrasted.